docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1325,8 +1278,8 @@ function clearMediaDeviceList() {
 }
 
 function getLocalMediaSettings() {
-  if (localMedia.cameraTrack) {
-    const videoSettings = localMedia.cameraTrack.getSettings();
+  if (localMedia.cameraStream) {
+    const videoSettings = localMedia.cameraStream.getTracks()[0]?.getSettings();
```

Comment:
Just a clarification. This getTracks() will go away right?

Reply:
We will take that as a Tech Debt after we release V3 I believe.

CC - @arun3528 @brycetham